lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "send_sms"
+require 'dotenv/load'
+
+class Order
+  def initialize(dish, quantity = 1)
+    @selected_dishes = []
+    quantity.times { @selected_dishes.push(dish) }
+    @confirmed = false
+    view_total
```

Comment:
Does this call the method at initialise? If so, is this to set up the @total variable?